src/ironoak/apriltag_rgb.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python3
+
```

Comment:
Can we move all the 4 files under `aiming` module? Later on, we can change it to specifically apriltag or tag_depth if that becomes the API.